src/javascript/app/pages/user/new_account/real_account_opening.js

```diff
@@ -104,7 +104,7 @@ const RealAccountOpening = (() => {
     const setPageTitle = () => {
         getElementById('page_title').innerHTML =
             real_account_signup_target === 'maltainvest'
-                ? localize('Financial Account Opening')
+                ? localize('Real money account opening')
```

Comment:
If it's the same then I think we don't need the `if else`

however we need to double confirm with the QA whom raised the issue